libraries/core_libs/network/rpc/eth/Eth.h

```diff
@@ -9,6 +9,7 @@ namespace taraxa::net::rpc::eth {
 struct EthParams {
   Address address;
   uint64_t chain_id = 0;
+  uint64_t gas_limit = ((uint64_t)1 << 53) - 1;
```

Comment:
Should be overwritten with dag block limit. So maybe better idea to have 0 default value. It would be easier to notice if we are not setting it somewhere